.subjects/STUD_PART/exam_02/2/tab_mult/tester.sh

```diff
@@ -3,10 +3,10 @@
 #                                                         :::      ::::::::    #
 #    tester.sh                                          :+:      :+:    :+:    #
 #                                                     +:+ +:+         +:+      #
-#    By: jcluzet <jcluzet@student.42.fr>            +#+  +:+       +#+         #
+#    By: nnuno-ca <nnuno-ca@student.42porto.com>    +#+  +:+       +#+         #
```

Comment:
You two should turn off your automatic header plugin. Otherwise I'd say you should keep jcluzet's header.